src/Tables.jl

```diff
@@ -32,6 +32,7 @@ Interface definition:
 | `Tables.columnnames(table)`                              | propertynames(table)        | Return column names for a table as an indexable collection                                                                                                   |
 | **Optional methods**                                     |                             |                                                                                                                                                              |
 | `Tables.getcolumn(table, ::Type{T}, i::Int, nm::Symbol)` | Tables.getcolumn(table, nm) | Given a column eltype `T`, index `i`, and column name `nm`, retrieve the column. Provides a type-stable or even constant-prop-able mechanism for efficiency. |
+| `Tables.ncols(table)`                                    | Tables.ncol(table)          | Return the number of columns                                                                                                                                 |
```

Comment:
```suggestion
| `Tables.ncol(table)`                                    | Tables.ncol(table)          | Return the number of columns                                                                                                                                 |
```
and the same change also in other spots

Reply:
Well spotted. Fixed in f094263